packages/page-staking/src/Nomination/NominatedAccount.tsx

```diff
@@ -0,0 +1,350 @@
+                  isPrimary={false}
+                  label={'Stop'}
+                  icon='stop'
+                  key='stop'
+                  tx='staking.chill'
+                />
+              )}
+              <TxButton
+                isDisabled={!isOwnController || isStashNominating}
+                accountId={controllerId}
+                isPrimary
+                label={t('Unbond')}
+                icon='sign-out'
+                params={[maxUnBond]}
+                tx='staking.unbond'
+                withSpinner={false}
+              />
+              <Popup
+                key='settings'
+                onClose={toggleSettings}
+                open={isSettingsOpen}
+                position='bottom right'
+                trigger={
+                  <Button
+                    icon='setting'
+                    onClick={toggleSettings}
+                  />
+                }
+              >
+                <Menu
+                  vertical
+                  text
+                  onClick={toggleSettings}
+                >
+                  {api.query.staking.activeEra && (
+                    <Menu.Item
+                      disabled={payoutEras.length === 0}
+                      onClick={_doPayout}
+                    >
+                      <Trans i18nKey='payoutEras'>
+                        {t('Payout reward')}&nbsp;{
+                        payoutEras.length
+                          ? <>(<FormatBalance value={payoutTotal} />)</>
+                          : ''
+                      }
+                      </Trans>
+                    </Menu.Item>
+                  )}
+                  <Menu.Item
+                    disabled={!isOwnStash && !balancesAll?.freeBalance.gtn(0)}
+                    onClick={toggleBondExtra}
+                  >
+                    {t('Bond more funds')}
+                  </Menu.Item>
+                  <Menu.Divider />
+                  <Menu.Item
+                    disabled={!isOwnStash}
+                    onClick={toggleSetController}
+                  >
+                    {t('Change controller account')}
+                  </Menu.Item>
+                  <Menu.Item
+                    disabled={!isOwnController}
+                    onClick={toggleRewardDestination}
+                  >
+                    {t('Change reward destination')}
+                  </Menu.Item>
+                  {isStashValidating &&
+                  <Menu.Item
+                    disabled={!isOwnController}
+                    onClick={toggleValidate}
+                  >
+                    {t('Change validator preferences')}
+                  </Menu.Item>
+                  }
+                  <Menu.Divider />
+                  {!isStashNominating &&
+                  <Menu.Item
+                    disabled={!isOwnController}
+                    onClick={toggleSetSession}
+                  >
+                    {t('Change session keys')}
```

Comment:
So, played a bit, and this copy shows all this info - this is not applicable at all to nominations. 

![image](https://user-images.githubusercontent.com/1424473/80144549-02256e80-85af-11ea-93ef-3e854c07a1c3.png)

Will drop UX feedback elsewhere.